src/chains/urls.py

```diff
@@ -6,5 +6,5 @@
 
 urlpatterns = [
     path("", ChainsListView.as_view(), name="list"),
```

Comment:
It should be `"/"`

Reply:
I've tried that 😅 It gives the following warning:

>?: (urls.W002) Your URL pattern '/' [name='list'] has a route beginning with a '/'. Remove this slash as it is unnecessary. If this pattern is targeted in an include(), ensure the include() pattern has a trailing '/'.

Reply:
for `chains/` it's the same behaviour as I described in the PR:

- `/chains` redirects (`HTTP 301`)
- `/chains/` works

Reply:
My mistake, I've just checked it and was getting the same error, sorry